analyses/cell-type-neuroblastoma-04/scripts/03c_run-scanvi-label-transfer.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+
+# Script to perform label transfer using scANVI/scArches using the scANVI NBAtlas model on an ScPCA query object prepared with ./03b_prepare-scanvi-query.R
+# This scANVI/scArches tutorial was used to help structure this script: https://docs.scvi-tools.org/en/1.3.2/tutorials/notebooks/multimodal/scarches_scvi_tools.html
+
+import argparse
+import sys
+from pathlib import Path
+
+import anndata
+import pandas as pd
+from scipy.sparse import csr_matrix
+import scvi
+import torch
+
+# Define constants
+BATCH_KEY = "Sample"
```

Comment:
Is the correct batch the sample or the library?

Reply:
It's the library, but the name of the column is `Sample` to match the NBAtlas object. I will add a comment to clarify this.